web/src/main/kotlin/no/nav/su/se/bakover/web/features/SuUserFeature.kt

```diff
@@ -67,8 +74,24 @@ class SuUserFeature(configuration: Configuration) {
     }
 }
 
+internal abstract class SuUserFeaturefeil(override val message: String) : Exception(message)
```

Comment:
Kunne kanskje extendet RuntimeException, men det spiller ingen rolle for Kotlin, siden den ikke har noe krav til å håndtere exception slik som java.